package.json

```diff
@@ -23,7 +23,7 @@
     "generate-types:spendingLimit": "typechain --target ethers-v5 --out-dir src/types/contracts ./node_modules/@gnosis.pm/safe-modules-deployments/dist/assets/**/*.json",
     "generate-types": "yarn generate-types:safeDeployments && yarn generate-types:spendingLimit",
     "postinstall": "yarn generate-types && yarn css-vars",
-    "analyze": "ANALYZE=true yarn build",
+    "analyze": "cross-env ANALYZE=true yarn build",
```

Comment:
also here?

Reply:
Windows doesn't inject env variables unless we use `cross-env`.